Packs/Core/Playbooks/playbook-NGFW_Scan_README.md

```diff
@@ -0,0 +1,81 @@
+This playbook handles external and internal scanning alerts.
+
+**Attacker's Goals:**
+
+Adversaries may attempt to get a listing of services running on remote hosts, including those that may be vulnerable to remote software exploitation. Methods to acquire this information include port scans and vulnerability scans using tools that are brought onto a system.
+
+**Investigative Actions:**
+
+Investigate the scanner IP address using:
+
+* IP enrichment:
+* NGFW Internal Scan playbook
+* Endpoint Investigation Plan playbook
+* Entity enrichment
+
+**Response Actions**
+
+The playbook's response actions are based on the initial data provided within the alert. In that phase, the playbook will execute:
+
+* Auto block IP address
+* Report IP address (If configured as true in the playbook inputs)
+
+When the playbook proceeds, it checks for additional activity using the Endpoint Investigation Plan playbook, and another phase, which includes the Containment Plan playbook, is executed.
```

Comment:
When the playbook proceeds, it checks for additional activity using the Endpoint Investigation Plan playbook, based on the data or the information found, the Containment Plan playbook will execute again.
In this phase, the playbook will execute the following containment actions:


I think that is clearer, wdyt?

Reply:
nope, I had something written like yours but Grammarly fixed it. now, I think the current is better.